src/Elastic.Apm.GraphQL.HotChocolate/HotChocolateDiagnosticListener.cs

```diff
@@ -18,6 +18,8 @@ internal class HotChocolateDiagnosticListener : DiagnosticEventListener
         private readonly IConfigurationReader _configuration;
         private readonly IApmLogger _apmLogger;
 
+        private readonly IActivityScope _emptyActivityScope = new EmptyActivityScope();
```

Comment:
remove and use EmptyScope property from Base class

Reply:
Fixed